trello_bot.py

```diff
@@ -0,0 +1,109 @@
+import os
+import requests
+from slack_sdk import WebClient
+from slack_bolt import App
+from dotenv import load_dotenv
+import re
+
+# Load environment variables from .env file
+load_dotenv()
+
+# Environment variables
+TRELLO_API_KEY = os.getenv('TRELLO_API_KEY')
+TRELLO_TOKEN = os.getenv('TRELLO_TOKEN')
+SLACK_BOT_TOKEN = os.getenv('SLACK_BOT_TOKEN')
+SLACK_APP_TOKEN = os.getenv('SLACK_APP_TOKEN')
+TRELLO_BOARD_ID = os.getenv('TRELLO_BOARD_ID')
+TRELLO_LIST_NAME = os.getenv('TRELLO_LIST_NAME')
+
+# Initialize the Slack app (Bolt framework)
+app = App(token=SLACK_BOT_TOKEN)
+
+# Fetch Trello list ID for the "To Do" list
+def get_trello_list_id():
+    url = f"https://api.trello.com/1/boards/{TRELLO_BOARD_ID}/lists?key={TRELLO_API_KEY}&token={TRELLO_TOKEN}"
+    response = requests.get(url)
+    
+    if response.status_code == 200:
+        lists = response.json()
+        for lst in lists:
+            if lst['name'].lower() == TRELLO_LIST_NAME.lower():
+                return lst['id']
+    raise Exception(f"Could not find list {TRELLO_LIST_NAME} on the board")
+
+# Add a card to the "To Do" list in Trello
+def add_card_to_trello(card_name):
+    list_id = get_trello_list_id()
+    url = f"https://api.trello.com/1/cards"
+    query = {
+        'key': TRELLO_API_KEY,
+        'token': TRELLO_TOKEN,
+        'idList': list_id,
+        'name': card_name
+    }
+    response = requests.post(url, params=query)
+    
+    if response.status_code == 200:
+        return response.json()
+    else:
+        raise Exception(f"Failed to add card to Trello: {response.status_code}, {response.text}")
+
+# Fetch Trello cards from the "To Do" list
+def get_trello_cards():
+    list_id = get_trello_list_id()
+    url = f"https://api.trello.com/1/lists/{list_id}/cards?key={TRELLO_API_KEY}&token={TRELLO_TOKEN}"
+    response = requests.get(url)
+    
+    if response.status_code == 200:
+        cards = response.json()
+        tasks = [card['name'] for card in cards]
+        return tasks
+    else:
+        raise Exception(f"Failed to fetch tasks from Trello: {response.status_code}, {response.text}")
+
+# Post a message to Slack channel
+def post_to_slack(channel, message):
```

Comment:
adding type hints can improve clarity regarding the expected input and output types.
For example:
def post_to_slack(channel: str, message: str) -> dict: